### ISSUE TITLE
Improve display for sequence tasks

### ISSUE THREAD

**Issue by nat-n** [nat-n/poethepoet#221]
#### Problems
1. Standard output with full task content is too verbose 
2. It's hard to follow what's happening with nested sequences

#### Ideas
1. Either make verbosity level 0 less verbose, or make -1 include (so -2 is required to have full silence)
2. Show when a task is part of a sequence e.g. `Poe => sequence-name[0] => task content`

**Comment by xixixao** [comment]
Related: We'd like to also hide the "command detail" output (what is poe running), but we'd like to keep the listing of commands when `poe` is run. This is not currently possible (verbosity -1 hides everything, 0 shows both).

**Comment by nat-n** [comment]
Hi @xixixao 

Could you provide a more specific example of what you mean? You'd like the CLI to just list the tasks, without detail of the task content (which can be quite verbose for shell tasks)?

**Comment by xixixao** [comment]
When I run `poe foo`, I want the command to just execute, and not print `Poe => <some details of foo definition>`.
When I run `poe`, I want the default output (which lists the available commands and their help messages).

**Comment by nat-n** [comment]
@xixixao thanks for pointing this out! I'd actually say this is a bug. I'd expect poe to ignore the default verbosity when running without a task :) I'll look into fixing this at the next opportunity :)
